src/graphic/graphic.cc

```diff
@@ -124,12 +124,21 @@ void Graphic::initialize(const TraceGl& trace_gl,
 
 	std::map<std::string, std::unique_ptr<Texture>> textures_in_atlas;
 	auto texture_atlases = build_texture_atlas(max_texture_size_, &textures_in_atlas);
-	image_cache_->fill_with_texture_atlases(
+	g_image_cache = new ImageCache();
+	g_image_cache->fill_with_texture_atlases(
 	   std::move(texture_atlases), std::move(textures_in_atlas));
-	styles().init();
+	g_style_manager = new StyleManager();
+	g_style_manager->init();
```

Comment:
Since you moved `StyleManager` out of the graphics class, the `init()` code can now be shifted into its constructor :)